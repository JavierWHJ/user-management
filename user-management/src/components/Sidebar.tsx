import React from 'react';
import { Nav } from 'react-bootstrap';
import '../App.css';

const Sidebar = () => {
    return (
        <Nav className="col-md-12 d-none d-md-block bg-light sidebar">
            <div className="container-fluid text-center d-flex flex-column align-items-center justify-content-center">
                <img
                    alt=""
                    src="/user.svg"
                    width="40"
                    height="40"
                    className="d-inline-block align-top"
                />
                <span className="my-3">Welcome User!</span>
            </div>
            <div className="sidebar-sticky"></div>
            <Nav.Item>
                <Nav.Link active>Users</Nav.Link>
            </Nav.Item>
            <Nav.Item>
                <Nav.Link eventKey="link-1">Tools</Nav.Link>
            </Nav.Item>
            <Nav.Item>
                <Nav.Link eventKey="link-2">Settings</Nav.Link>
            </Nav.Item>
        </Nav>
    );
}

export default Sidebar;
