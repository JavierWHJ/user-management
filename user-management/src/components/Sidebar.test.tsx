import React from "react";
import Sidebar from "./Sidebar";

import { render } from "@testing-library/react";

it("matches snapshot", () => {
  const { asFragment } = render(<Sidebar />);

  expect(asFragment()).toMatchSnapshot();
});