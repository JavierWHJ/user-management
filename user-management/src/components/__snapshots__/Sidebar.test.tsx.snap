// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="col-md-12 d-none d-md-block bg-light sidebar nav"
  >
    <div
      class="container-fluid text-center d-flex flex-column align-items-center justify-content-center"
    >
      <img
        alt=""
        class="d-inline-block align-top"
        height="40"
        src="/user.svg"
        width="40"
      />
      <span
        class="my-3"
      >
        Welcome User!
      </span>
    </div>
    <div
      class="sidebar-sticky"
    />
    <div
      class="ml-3"
    >
      <div
        class="nav-item"
      >
        <a
          class="nav-link active"
          href="#"
          role="button"
        >
          Users
        </a>
      </div>
      <div
        class="nav-item"
      >
        <a
          class="nav-link"
          data-rb-event-key="link-1"
          href="#"
          role="button"
        >
          Tools
        </a>
      </div>
      <div
        class="nav-item"
      >
        <a
          class="nav-link"
          data-rb-event-key="link-2"
          href="#"
          role="button"
        >
          Settings
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
